test/CMakeLists.txt

```diff
@@ -2,16 +2,16 @@ file(GLOB TESTS "test_*.c")
 
 include_directories("${CMAKE_CURRENT_SOURCE_DIR}/../src")
 
-foreach(test ${TESTS})
```

Comment:
I don't think the variable name here needs to change. Could you revert this change?